packages/oui-button/src/button.html

```diff
@@ -0,0 +1,20 @@
+<button
+  arial-label="{{$ctrl.text}}"
```

Comment:
I don't think aria-label will change after first digest, one time binding

Reply:
Yes, we could do that with the text too